integration-tests/src/test/scala/tech/beshu/ror/integration/suites/ReindexSuite.scala

```diff
@@ -40,13 +40,25 @@ trait ReindexSuite
   "A reindex request" should {
     "be able to proceed" when {
       "user has permission to source index and dest index"  in {
-        val result = user1ActionManager.actionPost("_reindex", ReindexSuite.reindexPayload("test1_index"))
+        val result = user1ActionManager.actionPost("_reindex", ReindexSuite.reindexPayload("test1_index", "test1_index_reindexed"))
         assertEquals(200, result.getResponseCode)
       }
     }
     "not be able to proceed" when {
-      "user has no permission to source index and dest index"  in {
-        val result = user1ActionManager.actionPost("_reindex", ReindexSuite.reindexPayload("test2_index"))
+      "user has no permission to source index and dest index which are present on ES"  in {
+        val result = user1ActionManager.actionPost("_reindex", ReindexSuite.reindexPayload("test2_index", "test2_index_reindexed"))
```

Comment:
let's refactor also this one:
* don't use actionPost (we'll remove ActionManagerJ in the future)
* add proper method (more specialised one) to IndexManager